coder-sdk/env.go

```diff
@@ -81,50 +84,39 @@ type CreateEnvironmentRequest struct {
 
 // CreateEnvironment sends a request to create an environment.
 func (c Client) CreateEnvironment(ctx context.Context, orgID string, req CreateEnvironmentRequest) (*Environment, error) {
-	var env *Environment
-	err := c.requestBody(
-		ctx,
-		http.MethodPost, "/api/orgs/"+orgID+"/environments",
-		req,
-		env,
-	)
-	return env, err
+	var env Environment
+	if err := c.requestBody(ctx, http.MethodPost, "/api/orgs/"+orgID+"/environments", req, &env); err != nil {
+		return nil, err
+	}
+	return &env, nil
 }
 
 // EnvironmentsByOrganization gets the list of environments owned by the given user.
 func (c Client) EnvironmentsByOrganization(ctx context.Context, userID, orgID string) ([]Environment, error) {
 	var envs []Environment
-	err := c.requestBody(
-		ctx,
-		http.MethodGet, "/api/orgs/"+orgID+"/members/"+userID+"/environments",
-		nil,
-		&envs,
-	)
-	return envs, err
+	if err := c.requestBody(ctx, http.MethodGet, "/api/orgs/"+orgID+"/members/"+userID+"/environments", nil, &envs); err != nil {
+		return nil, err
```

Comment:
should we be wrapping these as discussed?

Reply:
When there is only one error path, imho it is not necessary, but more is better, I'll add it.